### ISSUE TITLE
Specify version in file for Bower

### ISSUE THREAD

**Issue by Rendez** [rvera/image-picker#47]
rails-assets for example need a version specified (and/or tag) in order to build bower components.

**Comment by YPCrumble** [comment]
+1

**Comment by rvera** [comment]
Hi @YPCrumble, is this not fixed for you?

https://github.com/rvera/image-picker/tree/0.2.4

**Comment by YPCrumble** [comment]
@rvera having the same issue again. I looked deeper and found that the problem I'm facing is that the release is from January 2014, but the bower.json file was added later, in October. When Bower finds the package, it pulls the release, but not the bower.json file you added (which messes up other programs that are expecting to find that file). Solution would be to create a new release that will allow bower to include the bower.json file when it pulls that most recent release. See http://stackoverflow.com/questions/27384690/bower-install-save-is-not-creating-bower-json-file-from-bower-json for a good description.
